### ISSUE TITLE
[FEATURE REQUEST] docker and colab example 

### ISSUE THREAD

**Issue by mosheliv** [salesforce/Merlion#21]
**Is your feature request related to a problem? Please describe.**
Its difficult to try merlion because of requirements collision. A docker or maintained colab notebook would sove this nicely. Currently merlion installation in colab fas because of pandas/statsmodels collision. 

**Describe the solution you'd like**
A docker or colab notebook that just work. 

**Describe alternatives you've considered**

**Additional context**

**Comment by paulkass** [comment]
Could you elaborate on the colab pandas/statsmodels collisions that you're seeing? If I open a random colab notebook right now and try to play around with Merlion, am I bound to see these issues?

**Comment by mosheliv** [comment]
apologies, i should have included a notebook. here is one in a gist,
hopefully it opens well
https://gist.github.com/mosheliv/766e087ef327a38fd2c2ab33c430c4d1/raw/2b0a512a9d893d6b52d3eedf024500670db8c567/0_forecastintro.ipynb

**Comment by mosheliv** [comment]
just a few more comments...
I also tried it with my own dataset, no go, it fails later.
At least one easily runnable example will be much appreciated.

**Comment by mosheliv** [comment]
And the other problems were mostly with the nbeats notebook

**Comment by mosheliv** [comment]
I have managed to bypass this by cd to ts_datasets, which is not great.
However, the seatle trail dataset does not have a download function and so the nbeats example fails.

**Comment by mosheliv** [comment]
Note that this is not how your installation instructions go...
"You can install merlion from PyPI by calling pip install salesforce-merlion.
You may install from source by cloning this repo, navigating to the root
directory, and calling pip install ., or pip install -e . to install in
editable mode. You may install additional dependencies for plotting &
visualization via pip install salesforce-merlion[plot], or by calling pip
install ".[plot]" from the root directory of this repo.

To install the data loading package ts_datasets, clone this repo, navigate
to its root directory, and call pip install -e ts_datasets/. This package
must be installed in editable mode (i.e. with the -e flag) if you don't
want to manually specify the root directory of every dataset when
initializing its data loader."
I'll give it another try although I more or less went with another package
just because it was easier to use.

Basically, it would be really fantastic if you could add to the repository
a working example of the example notebooks on colab, as this is how people
test new packages nowadays. Dockerfile is also fine, just something that
works out of the box.
I'll try to re-try nbeats on colab and will let you know if I encountered
more problems.

**Comment by mosheliv** [comment]
If I'll get a colab notebook running well, I'll make a pull request. Same
with docker. Your product looks very robust but the proof is in the
pudding, as they say here.

**Comment by aadyotb** [comment]
PR #38 updates the installation instructions as discussed.
